### DISCUSSION TITLE
Theory block "Creating and manipulating ..." doesn't run properly

### DISCUSSION THREAD

**Discussion by Haagt** [ilabcode/pyhgf#195] [Q&A] [answered]
Hey all,
I've found the HGF tutorial and examples and so far it has worked really well in introducing me to the HGF. 
Unfortunately I can't make the theory block "Creating and manipulating networks of probabilistic nodes" run. 
This is the code which doesn't work properly:

import matplotlib.pyplot as plt
import numpy as np
import seaborn as sns
from pyhgf.model import Network, HGF
from pyhgf.typing import AdjacencyLists

This is the displayed error message:
"ImportError: cannot import name 'Network' from 'pyhgf.typing' (/usr/local/lib/python3.10/dist-packages/pyhgf/typing.py)"

As far as I can see in the API "Network" and "AdjacencyLists" are not included in the version of the current pyhgf. Could it be, that those two are not yet included in the currrent version or that they have been removed?
How can I fix this issue?

Thanks a lot in advance!
Kind regards, Thorben

**Comment by LegrandNico** [comment]
Hi @Haagt ,

Thank you for pointing this out here and in your email. This error comes from a new API for handling networks that was not included in the last version (but was online in the documentation). I just released a new version (0.0.19) that includes those changes, so it should work from here.

**Reply by Haagt** [reply]
Hi @LegrandNico,
thanks for the quick reply and the update.

Unfortunately there is a new error message on the same code.
This time it says:
"      2 
----> 3 from typing import Callable, Dict, Optional, Tuple, Union, override
      4 
      5 import jax.numpy as jnp

ImportError: cannot import name 'override' from 'typing' (/usr/lib/python3.10/typing.py)"

Can you help me out with this?
Thanks a lot!

**Reply by LegrandNico** [reply, marked as answer]
Hi @Haagt 

Yes I see, this was due to some incompatibilities with Python < 3.12, this should be fixed now 👍 Let me know if you experience any other errors
